bucket_99/rofi/specification

```diff
@@ -20,28 +20,34 @@ OPTIONS_STANDARD=	none
 
 LICENSE=		MIT:single
 LICENSE_FILE=		MIT:{{WRKSRC}}/COPYING
+LICENSE_TERMS=		single:{{WRKSRC}}/COPYING
+LICENSE_SCHEME=		solo
 
 FPC_EQUIVALENT=		x11/rofi
 
 BUILD_DEPENDS=		check:single:standard
 			flex:single:standard
 
-BUILDRUN_DEPENDS=	startup-notification:single:standard
+BUILDRUN_DEPENDS=	libxkbcommon:single:standard
+			startup-notification:single:standard
 			xorg-xcb-util:single:standard
 			xorg-xcb-util-wm:single:standard
 			xorg-xcb-util-xrm:single:standard
-			libxkbcommon:single:standard
 
 USES=			bison gmake libtool pkgconfig shebangfix
 GNOME_COMPONENTS=	cairo gdkpixbuf librsvg pango
 XORG_COMPONENTS=	xcb
+
 MUST_CONFIGURE=		gnu
 CONFIGURE_ENV=		LEX="{{LOCALBASE}}/bin/flex"
+
 SHEBANG_FILES=		script/get_git_rev.sh
 			script/rofi-sensible-terminal
-bash_CMD=		/bin/sh
+SHEBANG_LANG=		bash
+SHEBANG_OLD_BASH=	"/usr/bin/env bash"
+SHEBANG_NEW_BASH=	/bin/sh
 
 post-patch:
-# Fix rofi-theme-selector which assumes we have GNU sed
+	# Fix rofi-theme-selector which assumes we have GNU sed
 	@${REINPLACE_CMD} 's|$${SED} -i |$${SED} -i "" |' \
```

Comment:
Are you sure this post-patch is needed?  The GNU sed is the tool being used internally.